src/briefcase/config.py

```diff
@@ -587,8 +647,11 @@ def parse_config(config_file, platform, output_format, console):
         raise BriefcaseConfigError("No tool.briefcase section in pyproject.toml") from e
 
     # Merge the PEP621 configuration (if it exists)
+    project_config = pyproject.get("project", {})
+    if pep639_info := get_pep639_license_info(project_config, console=console, cwd=cwd):
+        pyproject["project"]["license"] = pep639_info
```

Comment:
Is this block needed? The global config will be processed later (currently L668) as part of the iteration over *all* project + app targets - won't having it here as well result in double processing?